.tool-versions

```diff
@@ -1,4 +1,4 @@
-elixir 1.13.4-otp-24
+elixir 1.14.3-otp-24
```

Comment:
need elixir 1.14, otherwise:

```
lib/ch/row_binary.ex:39: size in bitstring expects an integer or a variable as argument, got: :erlang.*(len, 8)
```

https://github.com/elixir-lang/elixir/pull/11558
https://github.com/erlang/eep/blob/master/eeps/eep-0052.md